x-pack/plugins/watcher/public/application/sections/watch_status_page/watch_status_page.tsx

```diff
@@ -77,23 +77,19 @@ export const WatchStatusPage = ({
 
   if (isWatchDetailLoading) {
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
+      <EuiPageTemplate.EmptyPrompt>
```

Comment:
Added `EuiPageTemplate.EmptyPrompt` otherwise the loading prompt doesn't have the color.
<img width="384" alt="Screenshot 2023-08-22 at 13 40 40" src="https://github.com/elastic/kibana/assets/6585477/fd361123-0447-4495-bc96-df8198431ce0">